lib/tasks/db.rake

```diff
@@ -89,6 +92,12 @@ end
   end
 
   def config
+    require 'db/connection'
```

Comment:
Maybe I'm misunderstanding something, but can we move this `require` to the top and just call `DB::Config.new` in the appropriate methods?